specification/gedcom-6-appendix-calendars.md

```diff
@@ -5,34 +5,42 @@
 This specification defines 4 calendars: `GREGORIAN`, `JULIAN`, `FRENCH_R`, and `HEBREW`.
 Previous versions also provided for, but did not define the meaning of, `ROMAN` and `UNKNOWN` calendars.
 
-Extension calendars should use the usual rules for extensions, including using `_` as the leading character of the calendar name. Month codes in extension calendars must either be already used for the same month name in another calendar or must start with `_`. Only upper case characters are allowed in month codes.
+Extension calendars should use the usual rules for extensions, including using `_` as the leading character of the calendar name.
 
 Each calendar must list its permitted epochs and their meaning.
 
+All month tags must either be standard tags defined for the month name in some standard calendar or be extension tags. Only upper case characters are allowed in month tags.
```

Comment:
```suggestion
All month tags must either be standard tags defined for the month name in some standard calendar or be extension tags.
```

The `Tag` production disallows lower-case characters so I don't think we need this.